apis/nucleus/src/components/listbox/__tests__/list-box-portal.test.jsx

```diff
@@ -131,6 +131,7 @@ describe('ListBoxPortal', () => {
             ...options,
             selections: options.selectionsApi,
             model: options.sessionModel,
+            app: options.app,
```

Comment:
```suggestion
```

Reply:
I think I made the test a bit easier to read.